### PR TITLE
git-svn: add example for making a shallow clone

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks for the review, @joelhy :)

File: pages/common/git-svn.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 - Clone an SVN repository:
 
-`git svn clone {{http://example.com/my_subversion_repo}} {{local_dir}}`
+`git svn clone {{http://example.com/subversion_repo}} {{local_dir}}`
+
+- Clone a SVN repository starting at a given revision number:
+
+`git svn clone -r{{1234}}:HEAD {{svn://svn.example.net/subversion/repo}} {{local_dir}}`
```
Comment (agnivade) [inline pages/common/git-svn.md]: Just curious - the previous example starts with "http://", but this has "svn://". Does this matter ? Or its intended ?

Comment (joelhy) [inline pages/common/git-svn.md]: It doesn't matter. SVN support both http and svn protocol. But better to use consistent url form.

Comment (waldyrious) [inline pages/common/git-svn.md]: I wanted to leave this using a different URL format precisely to demonstrate the variation that exists in the real world (this particular example is modeled after sourcefourge SVN URLs)

Comment (agnivade) [inline pages/common/git-svn.md]: Its a bit unclear whether one should use svn:// for this and http:// for the other, or they are interchangeable. A clarification would help here.

Comment (waldyrious) [inline pages/common/git-svn.md]: Yeah, I see what you mean. Let's use https everywhere then.

**Comment by agnivade** [comment]
LGTM

**Comment by waldyrious** [comment]
Cool, let's merge! :shipit:

**Pull request merged**